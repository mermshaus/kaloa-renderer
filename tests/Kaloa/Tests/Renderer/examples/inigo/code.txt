[code]1 > 2[/code]
[code=php]function f($x) {
    return $x * $x;
}[/code]
[code="php"]function f($x) {
    return $x * $x;
}[/code]


[code lang="php"]function f($x) {
    return $x * $x;
}[/code]

[code lang=php]function f($x) {
    return $x * $x;
}[/code]
