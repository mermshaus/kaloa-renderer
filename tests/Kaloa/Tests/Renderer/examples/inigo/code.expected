<pre><code class="language-none">1 &gt; 2</code></pre>

<pre><code class="language-php">function f($x) {
    return $x * $x;
}</code></pre>

<pre><code class="language-php">function f($x) {
    return $x * $x;
}</code></pre>

<pre><code class="language-php">function f($x) {
    return $x * $x;
}</code></pre>

<pre><code class="language-php">function f($x) {
    return $x * $x;
}</code></pre>